ghcide/test/exe/Main.hs

```diff
@@ -4781,11 +4781,8 @@ completionNoCommandTest name src pos wanted = testSession name $ do
   docId <- createDoc "A.hs" "haskell" (T.unlines src)
   _ <- waitForDiagnostics
   compls <- getCompletions docId pos
-  let wantedC = find ( \case
-            CompletionItem {_insertText = Just x} -> wanted `T.isPrefixOf` x
-            _                                     -> False
-            ) compls
-  case wantedC of
+  let isPrefixOfInsertOrLabel ci = any (wanted `T.isPrefixOf`) [fromMaybe "" (ci ^. Lens.insertText), ci ^. Lens.label]
```

Comment:
I made this helper function a little more lenient, but I think this is fine:

```haskell
CompletionItem
 { _label :: Text            -- ^ The label of this completion item. By default also
                             -- the text that is inserted when selecting this
                             -- completion.
 , _insertText :: Maybe Text -- ^ A string that should be inserted a
                             -- document when selecting this completion.
                             -- When `falsy` the label is used.
 -- ...
 }
```

Reply:
Yeah, that seems reasonable.